beacon-chain/rpc/lookup/blocker.go

```diff
@@ -148,13 +147,13 @@ func (p *BeaconDbBlocker) Block(ctx context.Context, id []byte) (interfaces.Read
 //     we are technically not supposed to import a block to forkchoice unless we have the blobs, so the nuance here is if we can't find the file and we are inside the protocol-defined retention period, then it's actually a 500.
 //   - block exists, has commitments, outside retention period (greater of protocol- or user-specified) - ie just like block exists, no commitment
 func (p *BeaconDbBlocker) Blobs(ctx context.Context, id string, indices []uint64) ([]*blocks.VerifiedROBlob, *core.RpcError) {
-	var root []byte
+	var rootSlice []byte
```

Comment:
Why are we renaming this variable to make it longer? We already know that it is a slice based on its type.

Reply:
I see we are reserving `root` for the fixed size val inside the func. You can ignore this.

Reply:
because I have a `root` variable below that is of type `[32]byte`. I can maybe leave this one as `root` and rename the other one to `blockRoot`? Or rename this one to `rootBytes`?